distrax/_src/distributions/gamma.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2021 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Gamma distribution."""
+
+from typing import Tuple, Union
+
+import chex
+from distrax._src.distributions import distribution
+from distrax._src.utils import conversion
+import jax
+import jax.numpy as jnp
+from tensorflow_probability.substrates import jax as tfp
+
+tfd = tfp.distributions
+
+Array = chex.Array
+Numeric = chex.Numeric
+PRNGKey = chex.PRNGKey
+
+
+class Gamma(distribution.Distribution):
+  """Gamma distribution with parameters `concentration` and `rate`.
+  """
```

Comment:
nit: Please move the quotation marks to the end of the line above, to make it in 1 line